src/main/java/com/haem/blogbackend/service/CategoryService.java

```diff
@@ -0,0 +1,38 @@
+package com.haem.blogbackend.service;
+
+import com.haem.blogbackend.domain.Category;
+import com.haem.blogbackend.dto.request.CategoryCreateRequestDto;
+import com.haem.blogbackend.dto.response.CategoryResponseDto;
+import com.haem.blogbackend.repository.CategoryRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class CategoryService {
+    private final CategoryRepository categoryRepository;
+
+    public CategoryService(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public long getCategoryCount(){
+        return categoryRepository.count();
+    }
+
+    @Transactional
+    public CategoryResponseDto createCategory(CategoryCreateRequestDto requestDto){
+        Category category = new Category(requestDto.getCategoryName());
```

Comment:
정적 팩토리 메서드로 객체 생성해도 됨
```suggestion
Category category = Category.create(requestDto.getCategoryName());
```